### ISSUE TITLE
Do you use L1 Loss in pixel2pixel model?

### ISSUE THREAD

**Issue by xml94** [HelenMao/MSGAN#5]
In your paper, there are two kinds of losses, original GAN loss and your proposed mode seeking loss.

I want to ask whether you use L1 loss in training pixep2pixel model.

It seems that L1 loss will fight with your mode seeking loss since L1 loss try to make every output same but mode seeking loss aims at push every output diffferent according the noise distance.

Your job is really nice.

**Comment by HelenMao** [comment]
Thanks for your attention.
Yes, to be consistent with the baseline, we do not modify any losses in the original Pix2Pix model and also incorporate L1 loss.
We think the L1 loss may help to preserve the most pixel level information but it indeed will have some conflicts with mode seeking loss.
In fact, we once tried to replace the L1 loss with feature matching loss using discriminator features, it may have fewer conflicts compared with L1 loss. However,  since L1 distance in mode seeking loss is the design choice, and to be consistent with the baseline models, we do not modify any other losses in the original models.

**Comment by xml94** [comment]
@HelenMao Thanks for your quick reply.

It means that you just put the mode seeking loss in the original mode such as pixel2pixel and bicycleGAN?

Anyway, I understand your idea. Good work!

**Comment by xml94** [comment]
To be honest, we can not know whether there are any other loss function just from your paper because you just give two losses. Maybe it is better to tell us that what kinds of losses have been used in your comparison experiment. I argue this is basic information.

**Comment by HelenMao** [comment]
Thanks for your suggestion, we have pointed out that we do not modify any loss functions and parameters of the original model when incorporating with mode-seeking loss.

**Comment by xml94** [comment]
@HelenMao  I see the sentence in appendix A.

Have you ever tried to do ablation comparison.
For example, compared with BicycleGAN, have you attempted to remove the two encoders in BicycleGAN and then add the awesome idea of mode seeking. In this way, we can learn the different effectivenesses between the original BicycleGAN and mode seeking.

**Comment by HelenMao** [comment]
As mentioned in appendixes, for the fair comparison with BicycleGAN,  we use the same architecture of BicycleGAN for Pix2Pix model and Pix2Pix model with mode-seeking loss.  You can see all the comparisons in the paper and the appendixes.

**Comment by xml94** [comment]
Ok, thanks very much
